pvlib/tests/test_conftest.py

```diff
@@ -19,3 +20,24 @@ def test_fail_on_pvlib_version_pass():
 @fail_on_pvlib_version('100000.0')
 def test_fail_on_pvlib_version_fail_in_test():
     raise Exception
+
+
+# set up to test passing arguments to conftest.fail_on_pvlib_version
+@pytest.fixture(scope='function')
+def some_data():
+    return "some data"
+
+
+def alt_func(*args):
+    print(args[0])
+    return
+
+
+deprec_func = deprecated('0.8', alternative='alt_func',
+                         name='deprec_func', removal='0.9')(alt_func)
+
+
+@fail_on_pvlib_version('0.9', some_data)
+def test_deprecated_09(some_data):
+    with pytest.warns(pvlibDeprecationWarning):
+        deprec_func(some_data)
```

Comment:
```suggestion
        assert some_data == deprec_func(some_data)
```

Reply:
Now I understand my previous suggestion would not work. I think putting the assert here too would be helpful: `assert "some data" == deprec_func(some_data)`